projects/valtimo/access-control/src/lib/access-control.module.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2015-2020 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {CommonModule} from '@angular/common';
+import {NgModule} from '@angular/core';
+import {AccessControlOverviewComponent} from './components/access-control-overview/access-control-overview.component';
+
+@NgModule({
+  declarations: [AccessControlOverviewComponent],
```

Comment:
Why export a non-functional module and a non-functional component from the new library? Exporting the models and services seems enough to me